pkg/traceql/storage.go

```diff
@@ -26,6 +26,18 @@ func SearchMetaConditions() []Condition {
 	}
 }
 
+func ExemplarMetaConditions() []Condition {
+	// TODO: Configurable?
+	return []Condition{
+		{NewIntrinsic(IntrinsicTraceID), OpNone, nil},
+		{NewIntrinsic(IntrinsicSpanID), OpNone, nil},
+		{NewIntrinsic(IntrinsicTraceDuration), OpNone, nil},
+		{NewIntrinsic(IntrinsicTraceStartTime), OpNone, nil},
+		{NewIntrinsic(IntrinsicTraceRootService), OpNone, nil},
+		{NewIntrinsic(IntrinsicTraceRootSpan), OpNone, nil},
```

Comment:
This is adding a lot of extra work to metric queries. We should try to quit these iterators as soon as we have enough exemplars.

Reply:
Do we use all of these columns for exemplars? I was expecting trace ID, maybe span ID, combined with the existing span time, is enough to create the exemplar.   Agree, span ID will add significant overhead.  Would be interested to see the impact with and without loading it.

Reply:
yeah, that is quite a lot to pull and will have a large perf impact. i think we either need to seriously reduce that list (at the expense of a less impressive frontend experience) or pull less data.

was chatting with @mapno earlier. since we order our files by traceid we can probably just check the first page of each job (or even less) and get exemplars well distributed across the time range.

Reply:
Reduced to only trace ID for now.